src/dists.ad.js

```diff
@@ -1322,6 +1322,10 @@ var Marginal = makeDistributionType({
     this.supp = _.map(this.params.dist, function(obj) {
       return obj.val;
     });
+
+    this.getDist = function() {
```

Comment:
I was a little confused by the name `getDist` at first. I assumed that, following our new-ish nomenclature, `dist` here is an instance of our distribution type -- the thing formerly known as `erp`. It seems desirable to reserve the word `dist` in the codebase for this type wherever possible. Is there a clearer name for what happens here? Maybe `toObject`?

That said, I see that the distribution parameter is also called `dist`, which may have motivated this method name. There, it's less clear that there is a good replacement, and then there are a bunch of other places (`printDist`, `_cacheddist `) where it's unclear what the alternative would be, and having inconsistent names may be worse than the status quo? I don't know. Feel free to leave it as it is.